scripts/zones/Port_San_dOria/npcs/Fontoumant.lua

```diff
@@ -1,117 +1,53 @@
 -----------------------------------
 -- Area: Port San d'Oria
---  NPC: Fontoumant
--- Starts Quest: The Brugaire Consortium
+-- NPC: Fontoumant
 -- Involved in Quests: Riding on the Clouds
 -- !pos -10 -10 -122 232
 -----------------------------------
 local ID = require("scripts/zones/Port_San_dOria/IDs");
 require("scripts/globals/settings");
 require("scripts/globals/keyitems");
-require("scripts/globals/titles");
 require("scripts/globals/quests");
+
+local quests =
+{
+    require("scripts/quests/sandoria/the_brugaire_consortium")
+}
+quests = dsp.quest.involvedQuests(quests)
```

Comment:
instead of assigning quests and then reassigning it immediately, do you think it looks nicer or not to do 
```
local quests = dsp.quest.involvedQuests(
{
    require("scripts/quests/sandoria/the_brugaire_consortium")
})
```
?

Reply:
it probably looks more confusing at first but this is gonna be copy/pasted in a lot of places

Reply:
Hey, if project style were up to me, that opening table bracket would be on the same line as the opening paren~! :wink:

```
local quests = dsp.quest.involvedQuests({
    require("scripts/quests/sandoria/the_brugaire_consortium")
})
```

For your example, I guess there is a small concern that newer contributors might have a little difficulty making sure all parens/brackets for the construction call are there. But that's really of minor consideration in regards to template style. If it's just going to be copy-pasted, it probably won't pose a problem.

I have no preference either way~! Up to you!

Reply:
*eyes you like a monster*

Reply:
yeah, I like your style better, lets do that

then I think we're done with this and I'll merge it